style-editor.css

```diff
@@ -46,8 +50,15 @@ figcaption,
 
 .wp-block h4 {
 	color: var( --color-tertiary-text );
-	background-color: var( --color-highlight );
 	font-size: 1.75em;
+	line-height: 1.4;
+}
+
+.wp-block h4 > strong {
```

Comment:
This makes it so that the background is only applied when the text is also bold, which is not the default behavior when you add a heading block. Is that the intention, or just a way to get around displaying the h4 as `inline-block`?

Reply:
That's the intention, yeah. Making it appear as an `inline-block` breaks the layout and doesn't actually make the background color wrap correctly. Only way to do it is have an inline tag inside a block tag, and `strong` is the best way given it's all bold already.